javaslang/src/main/java/javaslang/Value.java

```diff
@@ -516,7 +516,7 @@ default CharSeq toCharSeq() {
      */
     @SuppressWarnings("unchecked")
     @GwtIncompatible("reflection is not supported")
-    default T[] toJavaArray(Class<T> componentType) {
+    default T[] toJavaArray(Class<? super T> componentType) {
```

Comment:
... or we can deprecate it and do it Java's way, i.e. providing a preinitialized array as param.